src/scf/self_consistent_field.jl

```diff
@@ -118,7 +122,7 @@ Solve the Kohn-Sham equations with a SCF algorithm, starting at ρ.
         info = merge(info, (energies=energies, ))
 
         # Apply mixing and pass it the full info as kwargs
-        δρ = mix_density(mixing, basis, ρout - ρin; info...)
+        δρ = mix_density(mixing, basis, ρout - ρin; occupation_threshold, info...)
```

Comment:
just put occupation_threshold into the info. It might be useful for other callbacks (e.g. the regular one where we do printing) as well.